azure/azure-release-pipeline.yml

```diff
@@ -22,12 +22,37 @@ pool:
 
 variables:
   - template: project.yml
+  - group: pact
+  - group: variable-group-read
 
 extends:
-  template: azure/common/apigee-deployment.yml@common
+  template: azure-release-template.yml
   parameters:
     service_name: ${{ variables.service_name }}
     short_service_name: ${{ variables.short_service_name }}
     service_base_path: ${{ variables.service_base_path }}
     product_display_name: ${{ variables.product_display_name }}
     product_description: ${{ variables.product_description }}
+    service_base_path_pr: none
+    spec_file: ${{ variables.spec_file }}
+    apigee_deployments:
+      - environment: internal-dev
+        proxy_path: live
+      - environment: internal-qa-sandbox
```

Comment:
Should we deploy to internal-dev-sandbox here as well?

Reply:
Could look into this now the environments are split, I believe there's still a race condition here in internal-dev, internal-dev-sandbox testing but will raise a ticket for investigation